integtest/src/main/java/org/corfudb/universe/node/Node.java

```diff
@@ -0,0 +1,40 @@
+package org.corfudb.universe.node;
+
+import java.time.Duration;
+
+/**
+ * Represent nodes within {@link org.corfudb.universe.service.Service}s of {@link org.corfudb.universe.cluster.Cluster}
+ */
+public interface Node {
+
+    /**
+     * Deploys a specific node into the cluster. Note that Node is immutable and changes to state will lead to returning
+     * a new immutable instance of the node.
+     *
+     * @return a new instance of node with the new state.
+     *
+     * @throws NodeException thrown when can not deploy {@link Node}
+     */
+    Node deploy() throws NodeException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.corfudb.universe.node.NodeException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)